adaptlearning/adapt_framework
# issue/1874: amalgamate assets folders

#1874 

#### moved
  adapt/css/fonts/ > fonts/
  adapt/css/assets/ > assets/
  adapt/css/adapt.min.css > adapt.min.css

#### added
  preprocessor for less, to change ``url(../../assets`` to ``url(assets/``
  outputs a warning for the user with files that need updating
![image](https://user-images.githubusercontent.com/7974663/35387512-f3d919fc-01c7-11e8-9d81-447323893da0.png)


#### to test
* clone adapt_framework
* install plugins
* build course
* check that all assets, css and fonts are displaying correctly
